kernel/ksym.c

```diff
@@ -78,3 +78,21 @@ void *ksym_id2addr(int symid)
 {
 	return __ksym_tbl[symid].addr;
 }
+
+static inline int strcmp(const char *l, const char *r)
```

Comment:
Remove it since we already have functional implementation in file `kernel/lib/strcmp.c`.